src/RLogin.tsx

```diff
@@ -42,6 +43,7 @@ export class RLogin {
   private userProviders: IProviderUserOptions[];
   private supportedChains?: number[];
   private backendUrl?: string;
+  // private dataVaultOptions?: DataVaultOptions
```

Comment:
This field is commented in many places, is it temporary?

Reply:
It is waiting for Data Vault web build. Opened https://github.com/rsksmart/rif-data-vault/issues/50